packages/target-electron/src/ipc.ts

```diff
@@ -173,6 +174,34 @@ export async function init(cwd: string, logHandler: LogHandler) {
     ev.returnValue = app.getPath(arg)
   })
 
+  ipcMain.handle('checkMediaAccess', (_ev, mediaType: string) => {
+    if (!systemPreferences.getMediaAccessStatus) {
+      return 'unknown'
+    }
+    if (mediaType === 'camera') {
+      return systemPreferences.getMediaAccessStatus('camera')
+    } else if (mediaType === 'microphone') {
+      return systemPreferences.getMediaAccessStatus('microphone')
+    } else {
+      log.warn('checkMediaAccess: unknown media type', mediaType)
+      return 'unknown'
+    }
+  })
+
+  ipcMain.handle('askForMediaAccess', (_ev, mediaType: string) => {
```

Comment:
Isn't this supposed to return `boolean`, according to `runtime.ts`?

Reply:
I changed the implementation. askForMediaAccess throws an error now instead return "unknown", when not supported on the current platform